### ISSUE TITLE
FIND install

### ISSUE THREAD

**Issue by mstormi** [openhab/openhabian#1532]
See https://community.openhab.org/t/find-install-issue/122800/1

Also, I just selected to install FIND on a fresh openHABian and it fails with

```
Executing: /tmp/apt-key-gpghome.JOsKKls4Mc/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys 04EE7237B7D453EC
gpg: keyserver receive failed: Server indicated a failure
```
@ecdye Ethan could you please have a look ?

**Comment by ecdye** [comment]
Yep, it might take a little while as I am really busy right now but I'll work on it.

**Comment by ecdye** [comment]
Fixed by 3a58732
